frontend/packages/dev-console/src/components/import/devfile/DevfileSampleInfo.tsx

```diff
@@ -20,7 +20,8 @@ const DevfileSampleInfo: React.FC<DevfileSampleInfoProps> = ({ devfileSample })
           <img
             className="co-catalog-item-icon__img co-catalog-item-icon__img--large"
             src={iconUrl}
-            alt={displayName}
+            alt=""
```

Comment:
I'm kinda surprised that's not a lint error. I can't think of a reason why we wouldn't want the dynamic alt text?

Reply:
Some of the components use alt="", some nothing, and this was the only one which use `alt={displayName}`.

As I also added role="none" to not render this for screen renders I don't think that an alt text brings any benefit here.

In worst case the screen reader (or other a11y) tools says the display name twice since this is what is render directly after the icon.

Reply:
Its maybe a good reason to use aria-hidden instead of role=none :thinking:

Reply:
Alt text for images is a bit different I think than _just_ a11y. It also manages a failure of URL in some browsers. So you get some text. I think alt attribute for images should _always_ be filled in at least in some fashion (being dynamic or descriptive of the image)

Reply:
The I would recommend that we set it to t("Icon") or displayName. Any preference?

Reply:
I guess displayName would work... probably a mix of the two would be better but I'm not sure we need to over think this.

Reply:
Updated to displayName and aria-hidden. PTAL. Thanks :)